code/mathematical-algorithms/coprime_numbers/coprime_numbers.js

```diff
@@ -0,0 +1,34 @@
+const Readline = require('readline');
```

Comment:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍